src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -219,6 +219,21 @@
                     "parameters": []
                 }
             ]
+        },
+        {
+            "path": "/v1/debug/restart_service",
+            "operations": [
+                {
+                    "method": "PUT",
+                    "summary": "Restart a redpanda service",
+                    "type": "void",
+                    "nickname": "restart_service",
+                    "produces": [
+                        "application/json"
+                    ],
+                    "parameters": []
```

Comment:
I think you need some parameters here:
```suggestion
                    "parameters": [
                    {
                    "name": "service",
                    "in": "query",
                    "required": true,
                    "type": "string"
                    }
                    ],
```

Reply:
You probably want to lift the format that was in `redpanda_services_restart.json`

Reply:
I have no explanation for this other than: this was a silly mistake.